Tests/CK.DB.Actor.Tests/GroupTests.cs

```diff
@@ -183,5 +186,32 @@ public void only_system_users_can_add_or_remove_users_from_group_System()
             }
         }
 
+        [Test]
+        public void sGroupUserAdd_should_throw_when_adding_an_actor_that_is_not_a_user()
+        {
+            var groupTable = TestHelper.StObjMap.StObjs.Obtain<GroupTable>();
+            var actorTable = TestHelper.StObjMap.StObjs.Obtain<ActorTable>();
+            Debug.Assert( groupTable != null, nameof( groupTable ) + " != null" );
+            Debug.Assert( actorTable != null, nameof( actorTable ) + " != null" );
+
+            using( var context = new SqlStandardCallContext() )
+            {
+                var groupId = groupTable.CreateGroup( context, 1 );
+
```

Comment:
Adds a comment:
// Directly call the sActorCreate procedure: it is not exposed on the C# side 
// since it there is no point to call it... except from tests.